charts/nri-kube-events/README.md

```diff
@@ -1,57 +1,149 @@
-| `podSecurityContext.runAsGroup`                            | runAsGroup GID for Pod Security Context.                                                                                                                                                                                                       | true                            |
-| `proxy`                                                    | Set proxy if you required as Kube-events-forwarder / infra-agent needs to send metrics to New Relic One.                                                                                                                                       |                                 |
-| `config`                                                   | Set config option in order to pass any configuration to the agent running as a sidecar. This object is map as a configuration map mounted in /etc/newrelic-infra.yml.                                                                          |                                 |
-| `deployment.annotations`                                   | The annotations to add to the `Deployment`.                                                                                                                                                                                                    | `{}`                            |
-| `customAttributes.*`                                       | Custom attributes/Extra Label that will be added to all the metrics sent by Kube events. Queryable in New Relic.                                                                                                                               | `{}`                            |
-
-## Example
-
-Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
-
-Then, to install this chart, run the following command:
-
-```sh
-helm install newrelic/nri-kube-events \
---set licenseKey=<enter_new_relic_license_key> \
---set cluster=my-k8s-cluster
+![Version: 2.2.2](https://img.shields.io/badge/Version-2.2.2-informational?style=flat-square) ![AppVersion: 1.8.0](https://img.shields.io/badge/AppVersion-1.8.0-informational?style=flat-square)
+
+A Helm chart to deploy the New Relic Kube Events router
+
+**Homepage:** <https://docs.newrelic.com/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration>
+
+## Source Code
+
+* <https://github.com/newrelic/nri-kube-events/>
+* <https://github.com/newrelic/nri-kube-events/tree/master/charts/newrelic-infrastructure>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://helm-charts.newrelic.com | common-library | 1.0.2 |
+
+## Values managed globally
+
+This chart implements the [New Relic's common Helm library](https://github.com/newrelic/helm-charts/tree/master/library/common-library) which
+means that is has a seamless UX between things that are configurable across different Helm charts.
```

Comment:
```suggestion
means that it has a seamless UX between things that are configurable across different Helm charts.
```

Reply:
Solved in [#9f9cfc0](https://github.com/newrelic/nri-kube-events/pull/112/commits/9f9cfc0e51242f18647739df110dcb9b8600a133)